includes/manifest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * HM Platform Plugin Manifest.
+ */
+
+namespace HM\Platform;
+
+// Cavalcade.
+Plugin::register( 'cavalcade', 'plugins/cavalcade/plugin.php' )
+      ->load_on( true )
```

Comment:
what is `load_on( true )`? I think we need to look at the naming to be a bit more obvious perhaps. `load_on` and `load_with`?

Reply:
It's in the docblock - effectively it's the trigger so when to load the plugin, could be an action name, or if it's something that returns true will run the loading code then. Could name it differently, guess I've been influenced by the JS syntax around events.

`load_with` is the callback the plugin file is loaded with. Defaults to a basic require of the file.